lib/installer.js

```diff
@@ -88,6 +88,7 @@ exports.getProtoc = getProtoc;
 function downloadRelease(version) {
     return __awaiter(this, void 0, void 0, function* () {
         // Download
+        version = denormalizeVersion(version);
```

Comment:
This breaks caching.

This line creates the cache using the "denormalized" version:

https://github.com/arduino/setup-protoc/blob/a97892a429d98fae78d26f40334ab7eb616d08b9/src/installer.ts#L114

But this line looks for an existing cache using the "normalized" version before downloading:

https://github.com/arduino/setup-protoc/blob/a97892a429d98fae78d26f40334ab7eb616d08b9/src/installer.ts#L56

This must be fixed by either switching to using the "denormalized" version in the `tc.find` call, or to using the "normalized" version in the `tc.cacheDir`. You can choose which you think is best.

Reply:
Whoops, thanks for catching this! It probably makes sense to cache with the "real" (i.e. denormalized) version, so I'll use that everywhere.